src/main/java/com/tstepnik/planner/service/TaskService.java

```diff
@@ -43,9 +48,17 @@ public Task getTask(Long taskId) {
 
     public Task addTask(Task task) {
         User user = authService.getLoggedUser();
+
+        LocalDateTime now = LocalDateTime.now().truncatedTo(ChronoUnit.SECONDS);
+
+        task.setCreationDate(now);
         if (task.getImportance() == null) {
             task.setImportance(DEFAULT_IMPORTANCE);
         }
+        if (task.getPlannedFor() == null) {
+            LocalDateTime defaultPlannedTime = LocalDateTime.of(LocalDate.now().plusDays(1), LocalTime.MIDNIGHT);
```

Comment:
If I am correct, if user adds task on monday planned will be tuesday midnight. Am I right?

Reply:
Yes.

Reply:
then it doesnt make sense. Why would it be that way? I though that if user doesnt give planned for date then we set it to same day, not next one.

Reply:
tuesday midnight is start of the day so it's like end of monday.